### DISCUSSION TITLE
我赞助了但是没有给我更新呀，而且群也没发给我

### DISCUSSION THREAD

**Discussion by AlphaNight001** [Meloong-Git/PCL#1813] [提问] [answered]
我赞助了但是没有给我更新呀，而且群也没发给我

**Comment by hejiehao** [comment]
私信更新+识别码

**Comment by WForst-Breeze** [comment, marked as answer]
相关问题请见[ PCL2 常见问题 兼 脑溢血急救中心](https://shimo.im/docs/qKPttVvXKqPD8YDC)

很可能是你已经触发了人工回复，在等待人工回复的过程中无法使用命令。
你如果只是想使用命令，可以发送【取消人工处理】并私信更新+识别码

**Comment by AlphaNight001** [comment]
没有用呀 我私信了龙腾猫跃   更新DC4C-045A-5020-A922  更新两个字放后面也没用  没用任何回复

风释清然SC ***@***.***> 于2023年5月7日周日 22:21写道：

**Comment by AlphaNight001** [comment]
我明白了


k Alpha ***@***.***> 于2023年5月7日周日 22:28写道：
